scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  specialty TEXT,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  appt_date  DATE,
+  appt_time TIME,
+  comments TEXT,
+  patient_id INTEGER REFERENCES patients(id),
+  doctor_id INTEGER REFERENCES doctors(id)
+);
+CREATE INDEX ON appointments(patient_id);
+CREATE INDEX ON appointments(doctor_id);
```

Comment:
Good but you don't need the last two lines CREATE INDEX, you already set up the REFERENCES.